Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.vision.examples;
+
+import androidx.annotation.NonNull;
+
+import com.mapbox.vision.ar.FenceVisualParams;
+import com.mapbox.vision.ar.LaneVisualParams;
+import com.mapbox.vision.ar.VisionArManager;
+import com.mapbox.vision.ar.core.models.Color;
+import com.mapbox.vision.ar.view.gl.VisionArView;
+
+public class ArCustomizationActivity extends ArActivity {
+
+    @Override
+    protected void setArRenderOptions(@NonNull final VisionArView visionArView) {
+        // Make sure lane is visible
+        visionArView.setLaneVisible(true);
+        // Set the desired lane length in meters.
+        VisionArManager.setLaneLength(40f);
+
+        // Create an instance of LaneVisualParams
+        final LaneVisualParams laneVisualParams = new LaneVisualParams(
+                // Set the color
+                new Color(1.0f, 0.0f, 0.0f, 1.0f),
+                // Set the width
+                1f,
+                // Set the length of the chevrons in meters
+                2.5f
+        );
+        // After `laneVisualParams` is configured you need to set it to `VisionArView`
+        visionArView.setLaneVisualParams(laneVisualParams);
+
+        // Enable fence rendering
+        visionArView.setFenceVisible(true);
+
+        // Create an instance of `FenceVisualParams`
+        final FenceVisualParams fenceVisualParams = new FenceVisualParams(
+                // Set the color of the fence
+                new Color(1.0f, 1.0f, 0.0f, 1.0f),
+                // Set the height of the fence in meters
+                2f,
+                // Set the vertical offset of the fence above the road surface in meters
+                1f,
+                // Set the horizontal offset of the fence from the center of the road in meters
+                3f,
+                // Set the number of arrows in the fence
+                3
+        );
+        // After `fenceVisualParams` is configured you need to set it to to `VisionArView`
+        visionArView.setFenceVisualParams(fenceVisualParams);
+        // Setting AR render overall quality a bit lower to gain performance
+        visionArView.setArQuality(0.8f);
+    }
+}
```

Comment:
What if add this code to ARActivity? Looks simple enough for understanding and IMHO doesn't deserve separate activity
I'd extracted lane customizaiton and fence customization to separate methods too

Reply:
Well, AR quality is part of AR customization talking strictly. So I'd leave it in activity showing capabilities of our AR customization.

Reply:
Agree with @kiryldz

Reply:
I mean not only quality but the whole code in method setArRenderOptions(). Sorry for bad code selection